community/py-pygit2/APKBUILD

```diff
@@ -2,15 +2,15 @@
 # Maintainer:
 pkgname=py-pygit2
 _pkgname=pygit2
-pkgver=0.27.1
+pkgver=0.27.2
 pkgrel=0
 pkgdesc="Python bindings for libgit2"
 url="https://github.com/libgit2/pygit2"
 arch="all"
 license="GPL-2.0-only-WITH-GCC-exception-2.0"
 depends="py-cffi py-six"
 makedepends="python2-dev python3-dev py-setuptools libgit2-dev
-	libssh2-dev libressl-dev"
+	libssh2-dev openssl-dev"
```

Comment:
Looks it can remove dependencies - with following patch I was able to build
```
diff --git a/community/py-pygit2/APKBUILD b/community/py-pygit2/APKBUILD
index ba3b245e79..1acd7f7129 100644
--- a/community/py-pygit2/APKBUILD
+++ b/community/py-pygit2/APKBUILD
@@ -9,8 +9,7 @@ url="https://github.com/libgit2/pygit2"
 arch="all"
 license="GPL-2.0-only-WITH-GCC-exception-2.0"
 depends="py-cffi py-six"
-makedepends="python2-dev python3-dev py-setuptools libgit2-dev
-       libssh2-dev openssl-dev"
+makedepends="python2-dev python3-dev py-setuptools libgit2-dev"
 subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
 source="https://pypi.io/packages/source/p/$_pkgname/$_pkgname-$pkgver.tar.gz"
 builddir="$srcdir/$_pkgname-$pkgver"
```

Reply:
I assume its needed to fetch repos over SSH. I just checked Arch and Debian, they include this dependency too.

Reply:
Do they include -dev or just add dependency on the packages?

Reply:
libssh and libssl are dependencies of libgit2, so I removed them.